pkg/manifests/config.go

```diff
@@ -52,7 +52,6 @@ const (
 type Config struct {
 	Images      *Images `json:"-"`
 	RemoteWrite bool    `json:"-"`
-	TechPreview bool    `json:"-"`
```

Comment:
We have [metrics-server PR](https://github.com/openshift/cluster-monitoring-operator/pull/2022) also using same config so I think we can still keep this config and drop the check only for collection profile?